File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    let sumEven = 0;
    let sumOdd = 0;
    for (let i = 0; i < num_list.length; i++) {
        if (i % 2 == 0) {
            sumEven += num_list[i];
        } else {
            sumOdd += num_list[i];
        }
    }
    return Math.max(sumEven, sumOdd);
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    sum_even = 0
    sum_odd = 0
    for i, num in enumerate(num_list):
        if i % 2 == 0:
            sum_even += num
        else:
            sum_odd += num
    return max(sum_even, sum_odd)